inst/templates/head.html

```diff
@@ -6,6 +6,9 @@
 
 <!-- jquery -->
 <script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha384-nrOSfDHtoPMzJHjVTdCopGqIqeYETSXhZDFyniQ8ZHcVy08QesyHcnOUpMpqnmWq" crossorigin="anonymous"></script>
+<script type="text/javascript">
+var jQuery_3_1_0 = $.noConflict(false);
```

Comment:
I think so because `pkgdown.js` comes before HTML dependencies so it wouldn't have been overridden yet. I also put it inside an anonymous function to scope it better.